Applications/VGG/res/vgg.ini

```diff
@@ -0,0 +1,228 @@
+
+# Layer Section : Name
+[conv2d_c1_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 64
+stride = 1,1
+padding = 1,1
+
+[conv2d_c2_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 64
+stride = 1,1
+padding = 1,1
+
+[pooling2d_p1_layer]
+Type=pooling2d
+pool_size = 2,2
+stride =2,2
+padding = 0,0
+pooling = max
+
+[conv2d_c3_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 128
+stride = 1,1
+padding = 1,1
+
+[conv2d_c4_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 128
+stride = 1,1
+padding = 1,1
+
+[pooling2d_p2_layer]
+Type=pooling2d
+pool_size = 2,2
+stride =2,2
+padding = 0,0
+pooling = max
+
+[conv2d_c5_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 256
+stride = 1,1
+padding = 1,1
+
+[conv2d_c6_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 256
+stride = 1,1
+padding = 1,1
+
+[conv2d_c7_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 256
+stride = 1,1
+padding = 1,1
+
+[pooling2d_p3_layer]
+Type=pooling2d
+pool_size = 2,2
+stride =2,2
+padding = 0,0
+pooling = max
+
+[conv2d_c8_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 512
+stride = 1,1
+padding = 1,1
+
+[conv2d_c9_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 512
+stride = 1,1
+padding = 1,1
+
+[conv2d_c10_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 512
+stride = 1,1
+padding = 1,1
+
+[pooling2d_p4_layer]
+Type=pooling2d
+pool_size = 2,2
+stride =2,2
+padding = 0,0
+pooling = max
+
+[conv2d_c11_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 512
+stride = 1,1
+padding = 1,1
+
+[conv2d_c12_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+Activation=relu
+weight_initializer = xavier_uniform
+filters = 512
+stride = 1,1
+padding = 1,1
+
+[conv2d_c13_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
+weight_initializer = xavier_uniform
+filters = 512
+stride = 1,1
+padding = 1,1
+
+[bn_normalization_b1_layer]
+Type = batch_normalization
+epsilon = 1.0e-6
+momentum = 0.9
+Activation=relu
+
+[pooling2d_p5_layer]
+Type=pooling2d
+pool_size = 2,2
+stride =2,2
+padding = 0,0
+pooling = max
+
+[flatten]
+Type=flatten
+
+[fc_f1_layer]
+Type = fully_connected
```

Comment:
for the cifar100, what will be the dimension hereafter flatten for a reference?

Reply:
the dimension will be 512 when it is flattened.